e2etests/report.js

```diff
@@ -0,0 +1,26 @@
+
+const report = require("multiple-cucumber-html-reporter");
+const fs = require("fs");
+
+report.generate( {
+    jsonDir: "./cypress/json-logs/",
+    reportPath : "./cucumber-report/",
+    metadata: {
+        browser: {
+            name :"chrome",
+            version: 60,
+        },
+        device: "local test machine",
```

Comment:
will it be local device ? and also the platform is fixed only to windows

Reply:
This will be addressed with github workflow